WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -42,13 +42,16 @@ final class OrdersTests: XCTestCase {
             .goBackToOrdersScreen()
     }
 
-    func test_edit_existing_order() throws {
+    func test_edit_existing_order_loads_correct_data() throws {
```

Comment:
```suggestion
    func test_load_existing_order() throws {
```

what do you think of updating test name to above? to match the naming convention of other tests where the verb is the second word in the test name.

Reply:
Yep, much better name! Thanks for catching that.